<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowingsystemClass</class>
 <widget class="QMainWindow" name="BorrowingsystemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2017</width>
    <height>1480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Borrowingsystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>幼圆</family>
          <pointsize>23</pointsize>
         </font>
        </property>
        <property name="text">
         <string>媒体借阅管理系统</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radioButton_book">
            <property name="text">
             <string>图书</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radioButton_video">
            <property name="text">
             <string>视频光盘</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="radioButton_picture">
            <property name="text">
             <string>图像</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>480</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>480</height>
           </size>
          </property>
         </widget>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="Additem">
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="deleteitem">
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="edititem">
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="searchitem">
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="saveitem">
             <property name="text">
              <string>存盘</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="searchitem_2">
             <property name="text">
              <string>读取</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QTableWidget" name="tableWidget"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2017</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Borrowingsystem.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>radioButton_book</sender>
   <signal>toggled(bool)</signal>
   <receiver>BorrowingsystemClass</receiver>
   <slot>click_book()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_video</sender>
   <signal>toggled(bool)</signal>
   <receiver>BorrowingsystemClass</receiver>
   <slot>click_video()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_picture</sender>
   <signal>toggled(bool)</signal>
   <receiver>BorrowingsystemClass</receiver>
   <slot>click_picture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1075</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>1036</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>click_additem()</slot>
  <slot>click_deleteitem()</slot>
  <slot>click_edititem()</slot>
  <slot>click_searchitem()</slot>
  <slot>click_book()</slot>
  <slot>click_video()</slot>
  <slot>click_picture()</slot>
 </slots>
</ui>
